src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public boolean getStatus() {
+        return isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getTaskType() {
+        return null;
+    }
+
+    public void printAddDetails(int taskCounter) {
+        System.out.println("The following task has been added:\n[" + getTaskType() +"][" + getStatusIcon() + "] " + getDescription());
```

Comment:
Please pay attention to line length. Try to keep line length shorter than 110 characters (soft limit). If the line exceeds the limit, use line wrapping at appropriate places of the line.